utils/service-ip/locals.tf

```diff
@@ -0,0 +1,4 @@
+locals {
```

Comment:
We can put locals in main.tf.
Pre-commit requires in a module these files: main.tf, variables.tf, outputs.tf, versions.tf and README.md (only a description is recommended in README.md. The documentation about Terraform code are generated).
It is recommaned also give an exemple how to use this module.